compiler/qsc_frontend/src/resolve.rs

```diff
@@ -108,6 +113,18 @@ pub(super) enum Error {
     #[diagnostic(help("this item is not implemented and cannot be used"))]
     #[diagnostic(code("Qsc.Resolve.Unimplemented"))]
     Unimplemented(String, #[label] Span),
+
+    #[error("this export is not a callable or type")]
+    #[diagnostic(code("Qsc.Resolve.ExportedNonItem"))]
+    ExportedNonItem(#[label] Span),
+
+    #[error("exporting external items is not yet supported")]
```

Comment:
I'm going to continue my crusade against the word "yet" in errors... 

```suggestion
    #[error("exporting external items is not supported")]
```